src/rust/src/x509/extensions.rs

```diff
@@ -52,9 +52,13 @@ pub(crate) fn encode_authority_key_identifier<'a>(
     } else {
         None
     };
+    let ka = cryptography_keepalive::KeepAlive::new();
```

Comment:
I don't think this is needed, you just need to change the scope of the `serial_bytes` local to be function, rather than block.

Reply:
yes, fixed!